.github/workflows/backend_unit_test.yml

```
@@ -0,0 +1,71 @@
+    runs-on: ubuntu-latest
+    services:
+      docker:
+        image: docker:19.03.12
+        options: --privileged
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # GitHubリポジトリの設定で定義した変数を.envファイルに書き込む
+      - name: Create .env file
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)